src/Manager.cc

```diff
@@ -1081,22 +1081,22 @@ void ManagerPrivate::LoadPlugin(const tinyxml2::XMLElement *_elem)
 #endif
   }
 
-  ignition::common::SystemPaths systemPaths;
-  systemPaths.SetPluginPathEnv("IGN_LAUNCH_PLUGIN_PATH");
-  systemPaths.AddPluginPaths(IGNITION_LAUNCH_PLUGIN_INSTALL_PATH);
+  gz::common::SystemPaths systemPaths;
+  systemPaths.SetPluginPathEnv("GZ_LAUNCH_PLUGIN_PATH");
```

Comment:
Needs tick-tock

Reply:
For context, how it works is:

`SetPluginPathsEnv` [sets an internal data class' string](https://github.com/gazebosim/gz-common/blob/d7266a3197ffc0876b54fac042412d78c4365816/src/SystemPaths.cc#L143)

And that string is used on [`FindSharedLibrary`](https://github.com/gazebosim/gz-common/blob/d7266a3197ffc0876b54fac042412d78c4365816/src/SystemPaths.cc#L161) in the [`PluginPaths()` constructor](https://github.com/gazebosim/gz-common/blob/d7266a3197ffc0876b54fac042412d78c4365816/src/SystemPaths.cc#L168), specifically [here](https://github.com/gazebosim/gz-common/blob/d7266a3197ffc0876b54fac042412d78c4365816/src/SystemPaths.cc#L152)

Reply:
> See: https://github.com/gazebosim/sdformat/blob/namespace_migration/usd/src/cmd/sdf2usd.cc#L180-L198

Ouch, that snippet from SDFormat needs to be rethought, it has too many assumptions that should be handled by `gz-fuel-tools` instead.